src/Ellipsis.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component, createRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { throttle } from 'lodash';
+
+class Ellipsis extends Component {
+    containerRef = createRef();
+
+    state = {
+        isPrepared: false,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.throttleHandleResize = throttle(this.handleResize, 250);
```

Comment:
I would add this throttle wait as a prop.

The default value can be 250, but we should provide a way to change it ;)